src/Core/Services/TypeHelper.cs

```diff
@@ -29,6 +35,7 @@ public static class TypeHelper
             [typeof(char)] = DbType.StringFixedLength,
             [typeof(Guid)] = DbType.Guid,
             [typeof(byte[])] = DbType.Binary,
+            [typeof(DateTimeOffset)] = DbType.DateTimeOffset,
```

Comment:
I wonder this was leftout as per this PR:
https://github.com/Azure/data-api-builder/pull/1442
for a reason.

Probably, we should NOT add it in this PR to avoid regression unless we know for sure - although its interesting to see all tests are passing.

Reply:
GraphQL Supported types tests failed when I included system types DateTime, DateTime2, and TimeSpan here. Those look to be handled by #1473  I reran those tests locally and they passed when i removed the entry you pointed out. So i'm removing to avoid any possible regression.

Reply:
removed. Running tests in ci/cd to confirm